app/src/main/java/org/p2p/wallet/infrastructure/network/interceptor/RpcInterceptor.kt

```diff
@@ -46,50 +44,12 @@ class RpcInterceptor(
 
     private fun createRpcRequest(chain: Interceptor.Chain): Request {
         val request = chain.request()
-
-        val json = getRequestJson(request)
         val url = request.url
-        val networkEnvironmentForRequest = getValidatedNetworkEnvironment(json)
         return request.newBuilder()
-            .url(createRpcUrl(url, networkEnvironmentForRequest))
+            .url(createRpcUrl(url, currentEnvironment))
             .build()
     }
 
-    private fun getRequestJson(request: Request): JSONObject? {
```

Comment:
why have you removed this one?

Reply:
it's have not usage

Reply:
it was used to determine environment for history requests, for now we don't use this requests, so there is no need in this function